examples/supp_plugin-rs/ta/src/main.rs

```diff
@@ -54,19 +54,24 @@ fn invoke_command(cmd_id: u32, params: &mut Parameters) -> Result<()> {
     trace_println!("[+] TA invoke command");
     let mut p0 = unsafe { params.0.as_memref().unwrap() };
     let inbuf = p0.buffer().to_vec();
-    trace_println!("[+] TA received value {:?} then send to plugin", p0.buffer());
+    trace_println!(
+        "[+] TA received value {:?} then send to plugin",
+        p0.buffer()
+    );
     let uuid = Uuid::parse_str(PLUGIN_UUID).unwrap();
 
     match Command::from(cmd_id) {
         Command::Ping => {
-            let mut plugin = LoadablePlugin::new(&uuid);
-            let outbuf = plugin.invoke(
-                PluginCommand::Print as u32, 
-                PLUGIN_SUBCMD_NULL, 
-                &inbuf
-            ).unwrap();
+            let plugin = LoadablePlugin::new(&uuid);
+            let outbuf = plugin
+                .invoke(PluginCommand::Print as u32, PLUGIN_SUBCMD_NULL, &inbuf)
+                .unwrap();
```

Comment:
It's a legacy coding style issue that we use `unwrap()` in many examples. Please help eliminate its usage here to prevent potential panics.